Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM debian:stretch-backports
+
+MAINTAINER Piotr Król <piotr.krol@3mdeb.com>
+
+ENV http_proxy ${http_proxy}
```

Comment:
@pietrushnic Can you elaborate on that one? Is the `http_proxy` required at container build time?
If you wanted to pass env variable from the host, I'm not sure if it would work this way. Above statement sets the `http_proxy` to empty string for for the build time.

Reply:
@macpijan this is not recent version of the script, I have to push fixes. I made a change and it look like that:

```
ARG HTTP_PROXY
ENV http_proxy ${HTTP_PROXY}
```

To jest po to aby użyć `apt-cacher`

Reply:
Yeah ARG would be the correct way to do that kind of stuff